website/docs/r/virtual_machine.html.markdown

```diff
@@ -172,6 +174,12 @@ A `boot_diagnostics` block supports the following:
 
 ---
 
+A `additional_capabilities` block supports the following:
+
+* `ultra_ssd_enabled` - (Required) Should Ultra SSD disk be enabled for this Virtual Machine?
```

Comment:
Could we add the instructions someone needs to follow to enabled this on their subscription here?

Reply:
The UltraSSD whitelisting is only a temporary measure since the feature is in preview.

Reply:
THe docs should contain all information required for a user to use this feature.

Reply:
@katbyte Sure, we can go ahead and get this added 
"Currently, ultra SSDs are in preview and you must enroll (https://aka.ms/UltraSSDPreviewSignUp)  in the preview in order to access them." 

One small problem though is that once this feature is GA, this doc would have to be updated again. Alternatively, we could update the above line and add that 'Once the feature is GA, enrolment would not be required to use this feature".

Also, got a confirmation that the subscription ID's you submitted have been whitelisted for UltraSSD. Can you please confirm if you are able to test now?

@venkey1000 Can you also add a reference the article 
https://docs.microsoft.com/en-us/azure/virtual-machines/windows/disks-enable-ultra-ssd